bundles/ng-table.js

```diff
@@ -710,7 +710,10 @@ function ngTableController($scope, NgTableParams, $timeout, $parse, $compile, $a
         ngTableEventsChannel.onPagesChanged(function (params, newPages) {
             $scope.pages = newPages;
             $timeout(function () {
-                $element[0].focus();
+                var currentEl = document.activeElement;
+                if (currentEl && currentEl.getAttribute('type') != "search") {
```

Comment:
I'm not sure this is the right way. There are other boxes that are used which are not type "search". Consider the filter boxes on a number of pages (see Administration->Common Objects->Security->Security Groups). Might it not be better to limit this to events that happen from the pagination controls? Perhaps checking to see if the activeElement is within the pagination container?

Reply:
It seems that at the time when this function is called, its already blurred and not focused on a paging button so I couldn't do that check. I've changed the logic to focus on the table in the case there isn't any thing currently focused and I haven't seen anything else trigger it other than the paging buttons.